blog/2020-02-19-debezium-camel-integration.adoc

```diff
@@ -0,0 +1,290 @@
+= Integration Scenarios with Debezium and Camel
+jpechane
+:awestruct-tags: [ camel, integration, quarkus ]
+:awestruct-layout: blog-post
+
+One of the typical Debezium uses cases is to use change data capture to integrate a legacy system with other systems in the organization.
+There are multiple ways how to achieve this goal
+
+* Write data to Kafka using Debezium and follow with a combination of Kafka Streams pipelines and Kafka Connect connectors to deliver the changes to other systems
+* Use Debezium Embedded engine in a Java standalone application and write the integration code using plain Java
+* Use existing integration framework or service bus to express the pipeline logic
+
+This article is focusing on the third option - a dedicated integration framework.
+
+
+== Apache Camel
+
+https://camel.apache.org/[Camel] is an open-source integration framework that enables the developer to read and write data from and to disparate systems and services.
+It provides a large amount of https://camel.apache.org/components/latest/[components] that either provide interfaces to 3rd party systems or provides an implementation of https://en.wikipedia.org/wiki/Enterprise_Integration_Patterns[enterprise integration patterns].
+
+This combination allows the developer to easily connect to target systems and express integration pipelines using declarative DSL.
+
+
+=== Camel and Debezium
+
+https://camel.apache.org/blog/Camel3-Whatsnew/[Camel 3] has been released by the end of the year 2019 and in addition to the major re-architecture new Debezium components has been added to the codebase.
+
+* https://camel.apache.org/components/latest/debezium-mysql-component.html[Debezium MySQL Connector Component]
+* https://camel.apache.org/components/latest/debezium-postgres-component.html[Debezium PostgreSQL Connector Component]
+* https://camel.apache.org/components/latest/debezium-sqlserver-component.html[Debezium SQL Server Connector Component]
+* https://camel.apache.org/components/latest/debezium-mongodb-component.html[Debezium MongoDB Connector Component]
+
+As you can see every Debezium connector is represented by its dedicated component.
+The advantage of this solution is the complete isolation of dependencies and the type-safe configuration of connector instances.
+
+Internally the component exposes a Debezium endpoint with Camel consumer that encapsulates an instance of https://debezium.io/documentation/reference/1.1/operations/embedded.html[Debezium embedded engine].
+
+
+== Example
+
+We have prepared https://github.com/debezium/debezium-examples/tree/master/camel-component[an example] of how to deploy an integration pipeline using Debezium component.
+
+The source of change data is a simple Question & Answer (QnA) application.
+The changes generated by the application are passed to the Camel pipeline that sends emails via an SMTP server and posts a tweet on a provided Twitter account.
```

Comment:
There should be a sentence on the nature of the changes (new answer records etc.)